codegen/src/main/resources/software/amazon/awssdk/codegen/rules/Outputs.java.resource

```diff
@@ -57,6 +64,10 @@ public class Outputs {
         if (dnsSuffix != null ? !dnsSuffix.equals(outputs.dnsSuffix) : outputs.dnsSuffix != null) {
             return false;
         }
+        if (implicitGlobalRegion != null ? !implicitGlobalRegion.equals(outputs.implicitGlobalRegion) :
```

Comment:
nit. the ternary condition inside the if condition seems a little convoluted. Could we break this out into 

```
if (implicitGlobalRegion != null && !implicitGlobalRegion.equals(outputs.implicitGlobalRegion) || 
     implicitGlobalRegion == null && outputs.implicitGlobalRegion != null) {
    return false;
}
```

let me know if you think this is easier

Reply:
yup easier to read, updated